sys/targets/targets.go

```diff
@@ -447,10 +465,17 @@ func initTarget(target *Target, OS, arch string) {
 		// https://github.com/google/syzkaller/pull/619
 		// https://github.com/google/syzkaller/issues/387
 		// https://github.com/google/syzkaller/commit/06db3cec94c54e1cf720cdd5db72761514569d56
-		target.CCompilerPrefix = ""
+		target.Triple = ""
+	}
+	if OS == "linux" && target.Triple != "" {
+		if target.cflagsCompiler == nil {
+			target.cflagsCompiler = make(map[string][]string)
+		}
+		target.cflagsCompiler["clang"] = append(target.cflagsCompiler["clang"], "--target="+target.Triple)
+		target.cflagsCompiler["clang"] = append(target.cflagsCompiler["clang"], "-ferror-limit=0")
```

Comment:
This whole block is quite bulky syntactically.
We could do the following below:
```
if os.Getenv("SYZ_CLANG") == "1" {
	target.CFlags = append(target.CFlags,
		"--target="+target.Triple,
		"-ferror-limit=0",  // because ...
	)
}
```
(looks more laconic and readable to me)